Client/Frontend/Strings.swift

```diff
@@ -821,6 +821,10 @@ extension String {
     
     // Widget - Shared
     public static let QuickActionsGalleryTitle = NSLocalizedString("TodayWidget.QuickActionsGalleryTitle", tableName: "Today", value: "Quick Actions", comment: "Quick Actions title when widget enters edit mode")
+   
+    
+    // Quick View - Gallery View
+    public static let QuickViewGalleryTile = NSLocalizedString("TodayWidget.QuickViewGalleryTitle", tableName: "Today", value: "Quick View", comment: "Quick View title user is picking a widget to add.")
```

Comment:
https://github.com/mozilla-mobile/firefox-ios/blob/main/Client/Frontend/Strings.swift#L871

I think its the same string as what we have in our main branch. Hopefully we are not adding it twice. 🤞